File: challenges/3-tipos-de-triangulo/python/edurs2602/triangulos.py
tri_lados = []


def tipoTriangulo(A, B, C):
    tri_lados.append(A)
    tri_lados.append(B)
    tri_lados.append(C)
    tri_lados.sort()

    A = tri_lados[2]
    B = tri_lados[1]
    C = tri_lados[0]

    if A >= B + C:
        print("NAO FORMA TRIANGULO")
    else:
        if A**2 == B**2 + C**2:
            print("TRIANGULO RETANGULO")
        if A**2 > B**2 + C**2:
            print("TRIANGULO OBTUSANGULO")
        if A**2 < B**2 + C**2:
            print("TRIANGULO ACUTANGULO")
        if A == B == C:
            print("TRIANGULO EQUILATERO")
        if A == B or B == C or C == A:
            print("TRIANGULO ISOSCELES")


A = float(input(""))
B = float(input(""))
C = float(input(""))
tri = tipoTriangulo(A, B, C)

File: challenges/3-tipos-de-triangulo/javascript/GKeslley/triangulos.js
const values = [process.argv[2], process.argv[3], process.argv[4]];
values.sort((a, b) => a - b).reverse();

const a = values[0];
const b = values[1];
const c = values[2];

if (a >= b + c) {
  console.log("NÃO FORMA TRIANGULO");
} else {
  if (a * a === b * b + c * c) {
    console.log("TRIÂNGULO RETANGULO");
  }
  if (a * a > b * b + c * c) {
    console.log("TRIÂNGULO OBSTUSANGULO");
  }
  if (a * a < b * b + c * c) {
    console.log("TRIÂNGULO ACUTANGULO");
  }
  if (a === b && b === c) {
    console.log("TRIÂNGULO EQUILATERO");
  } else if (a === b || b === c || a === c) {
    console.log("TRIÂNGULO ISOSCELES");
  }
}
